<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseUI</class>
 <widget class="QWidget" name="CourseUI">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton {
   background-color: rgb(97, 31, 31);
   border-radius: 8px;  
   color: #fff;
   font-family: Inter,sans-serif;
   font-size: 1.2rem;
   font-weight: 700;
   line-height: 1;
   margin: 0;
   outline: none;
   padding: 1rem 1.2rem;
   text-align: center;
   text-decoration: none;
   white-space: nowrap;
   border: 0;
 }
QPushButton:hover:!pressed
{
  background-color: rgb(100, 30, 30);
}
 QPushButton:hover {
     background-color: rgb(147, 81, 81);
 }</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="courseName">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>130</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>COURSES NAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="testFrame">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>80</y>
     <width>371</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgba(230, 230, 230, 255);
	border-radius: 8px;
}

QTextEdit {
	background-color: rgba(0, 0, 0, 0);
	border: 0px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="inforFrame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>831</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgba(230, 230, 230, 255);
	border-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Day modified:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rating:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="descriptionTxtBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>631</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="dayModifyLbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Day modified:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ratingLbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rating:</string>
    </property>
   </widget>
   <widget class="QLabel" name="authorLbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
